lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -833,11 +833,48 @@ CRI_EXPERIAN_KBV_M2B:
         ticfCriBeta:
           targetState: CRI_TICF_BEFORE_NO_MATCH
     enhanced-verification:
-      targetState: MITIGATION_02_OPTIONS_WITH_F2F_M2B
+      targetState: PYI_KBV_FAIL_M2B
       checkIfDisabled:
         f2f:
           targetState: MITIGATION_02_OPTIONS
 
+PYI_KBV_FAIL_M2B:
+  response:
```

Comment:
This still needs to be a mitigation route as before which I _think_ is just a case of adding `mitigationStart: true` here